monitoring/cluster-monitoring/examining-cluster-metrics.adoc

```diff
@@ -0,0 +1,17 @@
+[id="examining-cluster-metrics"]
+= Examining cluster metrics
+include::modules/common-attributes.adoc[]
+:context: querying-metrics
+
+toc::[]
+
+{product-title} 4 provides a Web interface to Prometheus, which enables you to run Prometheus Query Language (PromQL) queries and examine the metrics visualized on a plot. This provides an extensive overview of the cluster state and enables you to troubleshoot problems.
```

Comment:
Does it make sense to use {product-version} here?

Reply:
Yep, implemented.